activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-repo/src/main/java/org/activiti/cloud/services/query/app/repository/CustomizedTaskRepository.java

```diff
@@ -30,9 +32,11 @@
                                                 Pageable pageable);
 
     Iterable<TaskEntity> findInProcessInstanceScope(Predicate predicate);
-    
-    Page<TaskEntity> findInProcessInstanceScope(Predicate predicate, 
+
+    Page<TaskEntity> findInProcessInstanceScope(Predicate predicate,
                                                 Pageable pageable);
 
     boolean existsInProcessInstanceScope(Predicate predicate);
+
+    boolean isUserInvolvedInProcessInstanceTasks(String processInstanceId, String userId, List<String> userGroups);
```

Comment:
Do you really need to add a custom method?  I think you can use the method provided by `TaskRepository`  as `exists` or `count` passing your custom predicate. Also, we have a set of Tasks in [ProcessInstanceEntity](https://github.com/Activiti/activiti-cloud/blob/8b6584cf36c1efb6170fc3f8240b3216bf1c0028/activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-model/src/main/java/org/activiti/cloud/services/query/model/ProcessInstanceEntity.java#L126). Can they be returned to avoid executing a new database call?

Reply:
I mean the methods provided by Querydls, not `existsInProcessInstanceScope`